src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunTest.java

```diff
@@ -227,7 +227,7 @@ private void assertColor(WorkflowRun b, BallColor color) throws IOException {
         final WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
         final String groovy = "println 'hello'";
         try (ACLContext context = ACL.as(User.getById("dev", true))) {
-            p.setDefinition(new CpsFlowDefinition(groovy));
+            p.setDefinition(new CpsFlowDefinition(groovy, false /* for mock authorization strategy */));
```

Comment:
Maybe clearer to use an explanation like "for full script approval with non-privileged user"